gitgraph

```diff
@@ -12,6 +13,12 @@ QUIET=0
 
 ###############################################################################
 #
+
+TMP_HIGH="/tmp/$$-tmp.high"
+TMP_DOT="/tmp/$$-tmp.dot"
+TMP_SVG="/tmp/$$-out.svg"
+TMP_PNG="/tmp/$$-out.png"
+
```

Comment:
I would want to see my commits in the Git's history, even if edited after. Or parts (commits subsets) ofpull requests.